cayman/handle_args.py

```diff
@@ -45,11 +45,12 @@ def handle_args(args):
     subparsers = ap.add_subparsers(dest="command", required=True)
 
     annotate_proteome_ap = subparsers.add_parser("annotate_proteome", help="Annotate proteome with CAZy domains.")
-    annotate_proteome_ap.add_argument("cutoffs", type=str, help = "path to file containing HMM-specific p-value cutoffs")    
+    annotate_proteome_ap.add_argument("--cutoffs", type=str, help = "path to file containing HMM-specific p-value cutoffs")    
     annotate_proteome_ap.add_argument("hmmdb", type=str, help = "path to folder containing HMMs")
     annotate_proteome_ap.add_argument("proteins", type=str, help = "path to protein sequences in fasta format")    
```

Comment:
```suggestion
    annotate_proteome_ap.add_argument("hmmdb", type=str, help = "path to folder containing HMMs")
    annotate_proteome_ap.add_argument("proteins", type=str, help = "path to protein sequences in fasta format")    
    annotate_proteome_ap.add_argument("--cutoffs", type=str, help = "path to file containing HMM-specific p-value cutoffs")    

```

Reply:
otherwise we'll have argparser-problems again

Reply:
did you simply change the order of arguments? why was the previous version causing issues?

Reply:
In #7 we had the issue before with the order of named/positional arguments causing issues with calling cayman from the command line. I reckon this would cause a similar problem.

Reply:
I see, thanks!